meteor-dev/server/main.js

```diff
@@ -0,0 +1,14 @@
+import { Meteor } from 'meteor/meteor';
+import { ClientsCollection } from '../api/clients/clients';
+import '../api/clients/server/publications';
+import '../api/clients/server/methods';
+
+const yaml = require('js-yaml')
+const fs = require('fs')
+
+const CONFIG_FILE_VAR = process.env.CONFIG_FILE
```

Comment:
maybe do `process.env.CONFIG_FILE || "/etc/genisys.yaml"` so that the default is used if none was specified.

Reply:
Addressed in [6e1314b](https://github.com/xeluior/genisys/pull/88/commits/6e1314bf468eeece8a727ac4f4fb43defcd0d6b0)